client/components/deposits-overview/recent-deposits-list.tsx

```diff
@@ -5,6 +5,7 @@ import * as React from 'react';
 import { Flex, FlexItem, Icon } from '@wordpress/components';
 import { calendar } from '@wordpress/icons';
 import { Link } from '@woocommerce/components';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
```

Comment:
I may not be up to date with the discussion on the design, but should this be `notice-outline`?

Reply:
Yep I caught onto this and fixed it in 50359ddcf0205f28f90ebb2f22e7f25d5b7492ba. :)